cairo/tests/ethereum/cancun/vm/instructions/test_stack_instructions.py

```diff
@@ -0,0 +1,156 @@
+import pytest
+from ethereum_types.numeric import Uint
+from hypothesis import given
+
+from ethereum.cancun.vm.exceptions import ExceptionalHalt
+from ethereum.cancun.vm.instructions.stack import push_n
+from tests.utils.args_gen import Evm
+from tests.utils.strategies import evm_lite
+
+pytestmark = pytest.mark.python_vm
```

Comment:
do you need the python vm ?

Reply:
@enitrat mentioned he will need it in moving forward

Reply:
eni: everything must be run in python vm for now after #359 until we have the same API for dict_manager